requirements.txt

```diff
@@ -117,6 +114,28 @@ https://github.com/mathjazz/silme/archive/v0.9.3.zip#egg=silme==0.9.3 \
 # Requirements with sub-dependencies.
 # ==================================================================================
 
+requests==2.18.4 \
```

Comment:
Ideally this should be lower in the requirements file, in order to follow alphabetical order.

Reply:
cool, ill update